stbt_rig.py

```diff
@@ -1030,7 +1037,8 @@ def request(self, method, url, timeout=None, retry=None, **kwargs):
             try:
                 response = self._session.request(method, url, **kwargs)
                 # Success or 4xx client error: don't retry:
-                if response.status_code < 500:
+                if (response.status_code < 500 and
+                        response.status_code not in retry_codes):
```

Comment:
One issue is that it's not an error if we receive a 202: we don't want to sleep between requests because it's already long-poll.  I'll make this change